packages/cln_plugin/test/cln_plugin_manifest_test.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:cln_plugin/src/cln_plugin_base.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('Manifest test: ', () {
+    var plugin = Plugin(
+        dynamic: true,
+        onInit: (plugin) {
+          return Future.value({});
+        });
+    test('Property test - dynamic', () {
+      // We set dynamic as true in the plugin initialization, let's test the property.
+      expect(plugin.dynamic.runtimeType, bool);
+      expect(plugin.dynamic, true);
+    });
+    plugin.registerRPCMethod(
+        name: "foo",
+        usage: "this specifies the usage",
+        description: "an example of how register a RPC Method",
+        callback: (plugin, request) => Future.value(<String, Object>{
```

Comment:
I wish to add a unit test for the callback but I can't seem to figure out what test we should write for this.